src/main/java/core/basesyntax/service/StorageService.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service;
+
+import core.basesyntax.Storage;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.operation.BuyOperation;
+import core.basesyntax.operation.Operational;
+import core.basesyntax.operation.SupplyAndReturnOperation;
+import java.util.List;
+
+public class StorageService {
+    private Storage fruitStore;
+    private final Operational supplyAndReturn = new SupplyAndReturnOperation();
+    private final Operational buy = new BuyOperation();
+
+    public void storageWriter(String filePath) {
+        TransactionReader transactionReader = new TransactionReader();
+        fruitStore = new Storage();
+        List<Transaction> list = transactionReader.getTransactionList(filePath);
+
+        for (Transaction x : list) {
+            if (x.getType().equals("s") || x.getType().equals("r")) {
```

Comment:
can we use Map here?

Reply:
I think we can, but this concept does not fit in my head now, I need more time and practice to use it with understanding. You said that this solution is appropriate for this moment.

Reply:
good. If you have a questions - feel free to ask